content/docs/guides/pwa-amp/amp-in-pwa.md

```diff
@@ -125,3 +125,8 @@ You can see the "AMP in PWA" pattern in action in the [React sample](https://git
 * Grab the source code here: [https://github.com/ampproject/amp-publisher-sample/tree/master/amp-pwa](https://github.com/ampproject/amp-publisher-sample/tree/master/amp-pwa)
 * Use the React component standalone via npm: [https://www.npmjs.com/package/react-amp-document](https://www.npmjs.com/package/react-amp-document)
 * See it in action here: [https://choumx.github.io/amp-pwa/](https://choumx.github.io/amp-pwa/) (best on your phone or mobile emulation)
+
+You can also see a sample of PWA and AMP using Polymer framework. It uses [amp-viewer](amp-viewer) to embed AMP pages.
```

Comment:
Hi Barb, it does, you can check in [https://github.com/Polymer/news/blob/amp/src/news-article.html](https://github.com/Polymer/news/blob/amp/src/news-article.html)